dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,1097 @@
+        Log.getLogWriter().info("SS - staging row table " + stagingRowTable + " created successfully");
+        snc.sql("CREATE TABLE " + rowTable + " USING row OPTIONS() AS (SELECT CODE, DESCRIPTION FROM " + stagingRowTable + ")");
+        Log.getLogWriter().info("SS - row table " + rowTable + " created successfully");
+    }
+
+
+    public static void HydraTask_createRowColumnTableUsingDataFrame() {
+        HydraTask_initializeSnappyTest();
+        String hfile = snappyTest.getDataFrameDataLocation() + "/2015.parquet";
+        String reffile = snappyTest.getDataLocation() + "/airportcodeParquetData";
+        DataFrame airlineDataFrame, airlineRefDataFrame;
+        HashMap<String, String> props = new HashMap<String, String>();
+//        props.put("buckets","11");
+        airlineDataFrame = snc.read().load(hfile);
+        snc.read().load(hfile).registerTempTable(stagingColumnTable);
+        Log.getLogWriter().info("SS - airline dataframe loaded successfully");
+        snc.dropTable(columnTable, true);
+        snc.createTable(columnTable, "column", airlineDataFrame.schema(), toScalaMap(props));
+        airlineDataFrame.write().format("column").mode(SaveMode.Append).saveAsTable(columnTable);
+        Log.getLogWriter().info("SS - column table " + columnTable + " created successfully");
+        airlineRefDataFrame = snc.read().load(reffile);
+        snc.read().load(reffile).registerTempTable(stagingRowTable);
+        props.clear();
+        Log.getLogWriter().info("SS - airlineref dataframe loaded successfully");
+        snc.dropTable(rowTable, true);
+        snc.createTable(rowTable, "row", airlineRefDataFrame.schema(), toScalaMap(props));
+        airlineRefDataFrame.write().format("row").mode(SaveMode.Append).saveAsTable(rowTable);
+        Log.getLogWriter().info("SS - row table " + rowTable + " created successfully");
+    }
+
+    public static void HydraTask_dropRowColumnTable() {
+        snc.dropTable(stagingColumnTable, true);
+        snc.dropTable(columnTable, true);
+        snc.dropTable(stagingRowTable, true);
+        snc.dropTable(rowTable, true);
+    }
+
+    /**
+     * This method gives simple count(*) query results on airline table.
+     */
+
+    public static void HydraTask_countQueryOnColumnTable() {
+        String query1 = "SELECT count(*) FROM airline";
+        Row[] result = snc.sql(query1).collect();
+        Assert.assertEquals(1, result.length);
+        Long expectedValue = Long.valueOf(1000000);
+        Assert.assertEquals(expectedValue, ((GenericRow) result[0]).get(0));
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result[0]).get(0));
+    }
+
+
+    /**
+     * This method retrives which airline had the most flights each year.
+     */
+
+    public static void HydraTask_countQueryWithGroupByOrderBy() {
+        String query = "select  count(*) flightRecCount, description AirlineName, UniqueCarrier carrierCode ,yeari \n" +
```

Comment:
I thought the initial idea was to read these queries from the conf file from user to automate the things. like
Task HydraTask_countQueryWithGroupByOrderBy  
conf-file = """ 
expectedResult = someNumer